recipe/meta.yaml

```diff
@@ -13,32 +15,88 @@ source:
   sha256: 23793c8645480148e9559df96b386b780f92194c80120acce79fcdaae0d81f45
 
 build:
-  number: 0
-  script: python build/build.py {{ args }}                                          # [unix]
-  script: python build/build.py --make-tool=mingw32-make {{ args }}                 # [win]
+  number: 1
   script_env:
     - OS    # [win]
-  features:
-    - vc9   # [win and py27]
-    - vc14  # [win and py>=35]
 
 requirements:
   build:
     - toolchain
     - python
     - swig
+    - {{ compiler('cxx') }}  # [win]
     - m2w64-make  # [win]
 
 test:
   requires:
-    - python
+    - python *  # any python version is ok for sake of testing of shared libraries
```

Comment:
Sorry on mobile. I think this might only be one space, which is what the linter is hung up on.